src/core/Akka.Tests/Actor/Stash/StashCapacitySpecs.cs

```diff
@@ -26,80 +26,83 @@ public StashCapacitySpecs(ITestOutputHelper output) : base(Config.Empty, output:
     [Fact]
     public async Task ShouldGetAccurateStashReadingForUnboundedStash()
     {
-        var stashActor = Sys.ActorOf(Props.Create(() => new StashActor()));
-        stashActor.Tell(new StashActor.StashMessage("1"));
-        stashActor.Tell(new StashActor.StashMessage("2"));
-        stashActor.Tell(StashActor.GetStashReadout.Instance);
-        var readout1 = await ExpectMsgAsync<StashActor.StashReadout>();
+        // we're going to explicitly set the stash size here - this should be ignored since the stash type is unbounded
+        var stashActor = Sys.ActorOf(Props.Create(() => new UnboundedStashActor()).WithStashCapacity(10));
```

Comment:
Validate that `StashCapacity` gets ignored on actors programmed to use `IWithUnboundedStash` - this is the correct behavior.